independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/NestedMaps.java

```diff
@@ -38,6 +40,11 @@ public static void deepMerge(Map left, Map right) {
                 Map leftChild = (Map) left.get(key);
                 Map rightChild = (Map) right.get(key);
                 deepMerge(leftChild, rightChild);
+            } else if (right.get(key) instanceof Collection && left.get(key) instanceof Collection) {
+                Collection c = new ArrayList();
```

Comment:
If you're merging them, it would be nice to use a `LinkedHashSet` instead

Reply:
Ok done :)